plugins/reolink/src/main.ts

```diff
@@ -91,7 +91,7 @@ class ReolinkCamera extends RtspSmartCamera implements Camera, Reboot, Intercom,
         }
 
         try {
-            const ai: AIState = this.storageSettings.values.hasObjectDetector[0]?.value;
```

Comment:
I believe the plain hasObjectDetector can be checked now? I don't think the value property exists since its destructured in getAiState